### DISCUSSION TITLE
Generating time-based read models

### DISCUSSION THREAD

**Discussion by fbjerggaard** [Eventuous/eventuous#185] [Q&A] [answered]
In the documentation for Read models there is an example idea of creating a read model containing the "Cancellations for the last three months (per hotel)"

This would be quite useful in a project I'm working on, but I have a hard time figuring out how to do it with Eventuous - I don't assume I would need a cronjob running to clean it up?

**Comment by alexeyzimarev** [comment, marked as answer]
I don't recall such an example, but it might be done using the Connector with ElasticSearch target. Then, you can have a Cabana dashboard with such data. It's basically a query for cancellation events that are not older than three months. Depending on the use case, it might not require a projection as such.
